parkbenchcommon/ramdisk.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2018 Joel Allen Luellwitz and Emily Frost
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Provides a class for managing ramdisks."""
+
+__all__ = ['Ramdisk', 'RamdiskMountError', 'RamdiskOptionError']
+
+import logging
+import os
+import subprocess
+
+# This is easy to edit, just in case someone wants a disk measurable in terabytes.
+VALID_TMPFS_SIZE_SUFFIXES = ['K', 'k', 'M', 'm', 'G', 'g', '%']
```

Comment:
You tested both cases?

Reply:
Yes I did. Both cases result in the same size.